### PR TITLE
Latls inclavared

### PR DESCRIPTION
**Pull request by uudiin**
inclavared enable local report attestation feature

### CODE REVIEW

File: ra-tls/elv/ra-tls-client.c [hunk 1]
```diff
@@ -131,6 +152,8 @@ int ra_tls_send(int sockfd, const void *bufsnd, size_t sz_bufsnd, void *bufrcv,
         la_get_report_from_cert(der, derSz, &report);
         body = &report.body;
         printf("Local report verification\n");
+
+		g_eid = load_enclave();
```
Comment (YangLiang3) [inline ra-tls/elv/ra-tls-client.c:156]: please remove an indent

**Review by YangLiang3 — COMMENTED** [review]
other LGTM

File: ra-tls/Makefile [hunk 2]
```diff
@@ -78,13 +78,15 @@ wolfssl: WOLFSSL_CONFIGURE_FLAGS := --prefix=$(shell readlink -f $(PREFIX)) --en
...
-	if [ -d "./wolfssl" ]; then \
-		cd wolfssl && ./autogen.sh && CFLAGS="$(CFLAGS)" ./configure $(WOLFSSL_CONFIGURE_FLAGS) || true; \
-	fi
 	if [ ! -d "./wolfssl" ]; then \
-		git clone -b v4.6.0-stable https://github.com/wolfSSL/wolfssl && \
-		cd wolfssl && git apply ../patch/wolfssl.patch && \
-		./autogen.sh && CFLAGS="$(CFLAGS)" ./configure $(WOLFSSL_CONFIGURE_FLAGS) || true; \
+		git clone -b v4.6.0-stable https://github.com/wolfSSL/wolfssl || true; \
+	fi
+	if [ -d "./wolfssl" ]; then \
```
Comment (jiazhang0) [inline ra-tls/Makefile:84]: Why allowing the clone operation to go failure?
And if why checking whether ./wolfssl exists here?

Comment (uudiin) [reply ra-tls/Makefile:84]: git clone to allow failure is a mistake.

Mainly used when clone is needed for the first time.

Comment (uudiin) [reply ra-tls/Makefile:84]: > Why allowing the clone operation to go failure?

git clone to allow failure is a mistake.

> And if why checking whether ./wolfssl exists here? If not exist, the repo will be cloned, and then ./wolfssl will exist for sure, unless git clone failed.

Mainly used when clone is needed for the first time.

Comment (jiazhang0) [reply ra-tls/Makefile:84]: I mean this condition check `if [ -d "./wolfssl" ]; then` is not necessary?

Comment (uudiin) [reply ra-tls/Makefile:84]: Yes, this condition check can be deleted, I will remove it.

**Pull request merged**